system/datatypes/peer_reviewed_article.php

```diff
@@ -91,6 +91,8 @@ class peer_reviewed_article extends HyphaDatatypePage {
 	const CMD_DISCUSSION_CLOSED = 'discussion_closed';
 	const CMD_COMMENT = 'comment';
 
+	const APPROVES_NEEDED = 3;
```

Comment:
Perfect! Next step: make it configurable. :-)

Reply:
Yup :-)